core/test/cljstyle/format/indent_test.clj

```diff
@@ -7,24 +7,33 @@
 
 (defn reindent-string
   ([form-string]
-   (reindent-string form-string config/default-indents))
-  ([form-string indents]
+   (reindent-string form-string 2 config/default-indents))
+  ([form-string list-indent-size]
+   (reindent-string form-string list-indent-size config/default-indents))
+  ([form-string list-indent-size indents]
    (reformat-string form-string {:indentation? true
-                                 :indents indents})))
+                                 :indents indents
+                                 :list-indent-size list-indent-size})))
```

Comment:
Thanks for adding some new tests! The rest of the test code would have many fewer changes if you kept the two-argument version as-is accepting `indents` and used the three-argument version for the `list-indent-size` when you needed it. Seems like a lot of the existing tests just got a `2` inserted.